data/json/items/armor/belts.json

```
@@ -209,17 +210,18 @@
     "covers": [ "TORSO" ],
     "coverage": 20,
     "storage": "1 L",
-    "encumbrance": 10,
+    "encumbrance": 2,
     "material_thickness": 2,
     "use_action": {
       "type": "holster",
-      "holster_prompt": "Attach what to belt loop?",
-      "holster_msg": "You clip your %s to your %s",
-      "multi": 4,
-      "max_volume": "1 L",
-      "max_weight": 1000,
+      "holster_prompt": "Store tool or blade",
+      "holster_msg": "You put your %s in your %s",
```

```suggestion
      "holster_msg": "You put your %1$s in your %2$s",
```
This way translators can change the order of the format specifiers.